src/cache-manager/index.ts

```diff
@@ -433,14 +445,13 @@ export default class CacheManager {
     opts: { cacheResolve: DataCachedResolver, tag?: any },
     context: RequestContext,
   ): Promise<ResolveResult> {
-    const updatedCacheMetadata = this._updateCacheMetadata(ast, data, cacheMetadata, "subscription");
+    const updatedCacheMetadata = this._updateCacheMetadata(ast, data, context, cacheMetadata);
 
     (async () => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this function so that its implementation is not identical to the one on line 354. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS4144)